openfisca_us/variables/demographic/household.py

```diff
@@ -232,3 +232,10 @@ class is_homeless(Variable):
     definition_period = YEAR
     documentation = "Whether all members are homeless individuals and are not receiving free shelter throughout the month"
     label = "Is homeless"
+
+
+class household_size(Variable):
```

Comment:
At one point I thought this was unnecessary given `household.nb_persons()`, but given how much this simplifies testing (as shown in this PR) I think it's OK. @nikhilwoodruff do you agree?

If so though, these tests should use a `spm_unit_size` variable instead.

Reply:
Yeah I agree here